oracle_test.go

```diff
@@ -89,42 +89,65 @@ func TestOracle_Initialize(t *testing.T) {
 }
 
 func TestOracle_CreateUser(t *testing.T) {
-	connURL, cleanup := prepareOracleTestContainer(t)
-	defer cleanup()
+	type testCase struct {
```

Comment:
This part of the diff got mangled by git. This wraps the previous test in a table test and makes the creation statements configurable